cookbooks/bach_ambari/Berksfile

```diff
@@ -0,0 +1,17 @@
+source 'https://supermarket.chef.io'
+
+cookbook 'bcpc-hadoop', path: '../bcpc-hadoop'
+cookbook 'ambari', path: '../ambari'
+cookbook 'bcpc', path: '../bcpc'
+cookbook 'bach_krb5', path: '../bach_krb5'
+cookbook 'smoke-tests', path: '../smoke-tests'
+#cookbook 'locking_resource', path: '../locking_resource'
```

Comment:
This Berksfile needs some help, please chat with me tomorrow and I would be happy to offer advice. Please see [`cookbooks/bach_repository/Berksfile`](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bach_repository/Berksfile#L6-L7) for an example.